src/pages/recipes/containers/RecipeDetailsContainer.tsx

```diff
@@ -39,12 +42,16 @@ class RecipeDetailsContainer extends React.Component<Props> {
 
 const mapStateToProps = (state: any) => ({
   recipeDetails: state.recipe.recipeDetails,
+  onFavorite: state.likeRecipe,
```

Comment:
onFavorite is the likeRecipe function, you should pass a favorited boolean prop instead